expression/builtin_string.go

```diff
@@ -1979,25 +1979,38 @@ type toBase64FunctionClass struct {
 }
 
 func (c *toBase64FunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
-	sig := &builtinToBase64Sig{newBaseBuiltinFunc(args, ctx)}
+	bf, err := newBaseBuiltinFuncWithTp(args, ctx, tpString, tpString)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	bf.tp.Flen = base64NeededEncodedLength(bf.args[0].GetType().Flen)
```

Comment:
@XuHuaiyu En.. In MySQL `base64_encode_max_arg_length ` was used to avoid overflow. In 32bit platform is 0x5EC0D4C6 or 0x5EC0D4C77B03531BLL in 64bit platform. I don't know if it is right for TiDB. Could you give me some advise?